cmd/gator/expand/expand.go

```diff
@@ -59,18 +64,20 @@ func init() {
 	Cmd.Flags().StringArrayVarP(&flagFilenames, flagNameFilename, "n", []string{}, "a file or directory containing Kubernetes resources.  Can be specified multiple times.")
 	Cmd.Flags().StringVarP(&flagFormat, flagNameFormat, "f", "", fmt.Sprintf("Output format.  One of: %s|%s.", stringJSON, stringYAML))
 	Cmd.Flags().StringVarP(&flagOutput, flagNameOutput, "o", "", "Output file path. If the file already exists, it will be overwritten.")
+	Cmd.Flags().StringArrayVarP(&flagImages, flagNameImage, "i", []string{}, "a URL to an OCI image containing policies. Can be specified multiple times.")
```

Comment:
🚲 / UX:

here and for `test` : do we want to have some validation where a user of `gator` cannot specific a `flagTempDir ` without specifying an element on `flagImages`?

Reply:
If it's harmless to have it set, best not to add this constraint, otherwise user logic like the following would be harder to write:

* temp dir is /tmp
* Run validation of directories
* Optionally also include images

Users would need to add and/or remove the `temp dir` flag depending on whether images exist, when leaving it specified does no harm.